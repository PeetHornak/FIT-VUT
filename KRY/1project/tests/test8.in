Ose Wpcohzln dvygmvrek vce ezxbwmd ox ose kvxe xvxidt. Ehwdc swklrsop epddtwine an l mqos. Fgm dcazycw, hfsax, dpgme, elx, Pujjae mnps lcp sshp vgxlbmglrq. Ose dvygmvrek jylq ytfxzc if oseam rrshxaj, oseam argifnudltajy afy ehwdc mgne cghxof rzrvn. Pvwmjofz cesgtzwn hhq v yeo xzmejy lsirusbp wgpwd tz oekdcatgp: ofz nomgo rwafsw oz pst pxhzysaqp tjvysdveojn. Eo sxsiwqp tzdd, il rzudy me fzneknlrq oz hsqp ufdqojh rrshxaj, kcofpycaveigi lnv hzrw xzmejy wgmos. Aa denzcad glnyplgwn nosgpsuz, ehw bcaehlr ga ehw mpsmgeifb wafbfayz ts ejce kdxpdz lnv mpgmglr lcln lclt ga ehw dydaqtdmvw lsirusbps. Lcp nwr noehzn dvygmvre odwl tz xojz diekwe sio rwbflsm ehsi ehw ziikotny Zfrgkpaf glnyplgwn. Tt odwl tz ls kdxpdz ls Gxnivzytsg; tn xvnt, ao hidg me Gxnivzytsg. Eo si Pnygtsz kprkjy, il rtld npee gtkw ntmhgtfazo Efbwikc, ls s nvehotcsg Naewcivbp fjdpnv jq maip tggo mw rsal Jncaypnlvw ik. Ose Wpcohzln dvygmvrek vce ezxbwmd ox ose kvxe xvxidt. Ehwdc swklrsop epddtwine an l mqos. Fgm dcazycw, hfsax, dpgme, elx, Pujjae mnps lcp sshp vgxlbmglrq. Ose dvygmvrek jylq ytfxzc if oseam rrshxaj, oseam argifnudltajy afy ehwdc mgne cghxof rzrvn. Pvwmjofz cesgtzwn hhq v yeo xzmejy lsirusbp wgpwd tz oekdcatgp: ofz nomgo rwafsw oz pst pxhzysaqp tjvysdveojn. Eo sxsiwqp tzdd, il rzudy me fzneknlrq oz hsqp ufdqojh rrshxaj, kcofpycaveigi lnv hzrw xzmejy wgmos. Aa denzcad glnyplgwn nosgpsuz, ehw bcaehlr ga ehw mpsmgeifb wafbfayz ts ejce kdxpdz lnv mpgmglr lcln lclt ga ehw dydaqtdmvw lsirusbps. Lcp nwr noehzn dvygmvre odwl tz xojz diekwe sio rwbflsm ehsi ehw ziikotny Zfrgkpaf glnyplgwn. Tt odwl tz ls kdxpdz ls Gxnivzytsg; tn xvnt, ao hidg me Gxnivzytsg. Eo si Pnygtsz kprkjy, il rtld npee gtkw ntmhgtfazo Efbwikc, ls s nvehotcsg Naewcivbp fjdpnv jq maip tggo mw rsal Jncaypnlvw ik. Ose Wpcohzln dvygmvrek vce ezxbwmd ox ose kvxe xvxidt. Ehwdc swklrsop epddtwine an l mqos. Fgm dcazycw, hfsax, dpgme, elx, Pujjae mnps lcp sshp vgxlbmglrq. Ose dvygmvrek jylq ytfxzc if oseam rrshxaj, oseam argifnudltajy afy ehwdc mgne cghxof rzrvn. Pvwmjofz cesgtzwn hhq v yeo xzmejy lsirusbp wgpwd tz oekdcatgp: ofz nomgo rwafsw oz pst pxhzysaqp tjvysdveojn. Eo sxsiwqp tzdd, il rzudy me fzneknlrq oz hsqp ufdqojh rrshxaj, kcofpycaveigi lnv hzrw xzmejy wgmos. Aa denzcad glnyplgwn nosgpsuz, ehw bcaehlr ga ehw mpsmgeifb wafbfayz ts ejce kdxpdz lnv mpgmglr lcln lclt ga ehw dydaqtdmvw lsirusbps. Lcp nwr noehzn dvygmvre odwl tz xojz diekwe sio rwbflsm ehsi ehw ziikotny Zfrgkpaf glnyplgwn. Tt odwl tz ls kdxpdz ls Gxnivzytsg; tn xvnt, ao hidg me Gxnivzytsg. Eo si Pnygtsz kprkjy, il rtld npee gtkw ntmhgtfazo Efbwikc, ls s nvehotcsg Naewcivbp fjdpnv jq maip tggo mw rsal Jncaypnlvw ik. Ose Wpcohzln dvygmvrek vce ezxbwmd ox ose kvxe xvxidt. Ehwdc swklrsop epddtwine an l mqos. Fgm dcazycw, hfsax, dpgme, elx, Pujjae mnps lcp sshp vgxlbmglrq. Ose dvygmvrek jylq ytfxzc if oseam rrshxaj, oseam argifnudltajy afy ehwdc mgne cghxof rzrvn. Pvwmjofz cesgtzwn hhq v yeo xzmejy lsirusbp wgpwd tz oekdcatgp: ofz nomgo rwafsw oz pst pxhzysaqp tjvysdveojn. Eo sxsiwqp tzdd, il rzudy me fzneknlrq oz hsqp ufdqojh rrshxaj, kcofpycaveigi lnv hzrw xzmejy wgmos. Aa denzcad glnyplgwn nosgpsuz, ehw bcaehlr ga ehw mpsmgeifb wafbfayz ts ejce kdxpdz lnv mpgmglr lcln lclt ga ehw dydaqtdmvw lsirusbps. Lcp nwr noehzn dvygmvre odwl tz xojz diekwe sio rwbflsm ehsi ehw ziikotny Zfrgkpaf glnyplgwn. Tt odwl tz ls kdxpdz ls Gxnivzytsg; tn xvnt, ao hidg me Gxnivzytsg. Eo si Pnygtsz kprkjy, il rtld npee gtkw ntmhgtfazo Efbwikc, ls s nvehotcsg Naewcivbp


