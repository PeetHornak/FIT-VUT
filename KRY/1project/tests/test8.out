4;4;4;vlas
